packages/IMAPClient-UI.package/ICFolderDialog.class/instance/labelForMail..st

```diff
@@ -1,11 +1,13 @@
 email selection
 labelForMail: anICEmail
 
-	| fromCollection label |
+	| fromCollection subject label |
 
 	fromCollection := anICEmail from findBetweenSubStrs: '<>'.
 	
-	label := (anICEmail subject truncateTo: 80),
+	subject := anICEmail subject ifEmpty: [ICEmailDialog defaultSubject].
```

Comment:
Why not use `self subject`?

Reply:
`ICFolder` does not have `self subject`

I agree that it's not nice to have the default subject checks here and in the `ICEmail` as well, but I do not see a cleaner way to do that at the moment